src/global/BaseTest.java

```diff
@@ -208,7 +208,7 @@ public void outputFollowsCorrectStructure() {
         String output = getOutput();
         Matcher matcher = getMatches(output, processRegexForPrintlnOutput(combineRegex(getRegexSentence())));
         String incorrectOutputMessage = TestOption.incorrectStructureErrorMessage;
-       _assertTrue(matcher.find(), incorrectOutputMessage, parseTestInformation(output, getRegexSentence(), incorrectOutputMessage));
+        _assertTrue(matcher.find(), incorrectOutputMessage, parseTestInformation(output, getRegexSentence(), incorrectOutputMessage));
```

Comment:
This should not be in the PR

Reply:
Ok, should I make another PR? And if so what should I call it, since it's just such a small formatting change